bin/test-run

```diff
@@ -777,6 +780,9 @@ sub do_send_group_abuse {
     set_fyr_date('1990-08-26', undef, $verbose);
     call_fyrqd($wth, $verbose, $multispawn);
     foreach $rep_name (@new_rep_names){
+            # long lines are wrapped and our string matching is dumb so
+            # truncate the long name
+            $rep_name =~ s/Alex Hollingsworth/Alex/;
```

Comment:
Would it be better to replace the spaces with \s* so it matches across newlines, rather than hope no line breaks occur?